src/terminal.go

```diff
@@ -1758,6 +1759,46 @@ func (t *Terminal) executeCommand(template string, forcePlus bool, background bo
 	cleanTemporaryFiles()
 }
 
+func (t *Terminal) executeChangeQuery(template string) {
+	valid, list := t.buildPlusList(template, false)
+	if !valid {
+		return
+	}
+	command := t.replacePlaceholder(template, false, string(t.input), list)
+	cmd := util.ExecCommand(command, false)
+	newQuery := []rune{}
+	t.executing.Set(true)
+	cmd.Stdin = tui.TtyIn()
+	out, _ := cmd.StdoutPipe()
+	t.tui.Pause(false)
```

Comment:
I'm not entirely sure what `t.tui.Pause` and `t.tui.Resume` do, so I included them based on what `executeCommand` does. But they might not be actually needed.